.circleci/config.yml

```diff
@@ -94,6 +94,12 @@ jobs:
           shell: bash.exe
           command: .circleci/windows-node-upgrade.sh lts
 
+      # After upgrading the node version in a Windows executor, we have to manually reinstall Yarn if we want to use it.
```

Comment:
Could these be refactored into a `reinstall_yarn` subtask?

Reply:
They could. I'll do that.